caravel/assets/javascripts/reduxUtils.js

```diff
@@ -67,9 +67,8 @@ export function addToArr(state, arrKey, obj) {
 export function enhancer() {
   let enhancerWithPersistState = compose(persistState());
   if (process.env.NODE_ENV === 'dev') {
-    enhancerWithPersistState = compose(
-      persistState(), window.devToolsExtension && window.devToolsExtension()
-    );
+    const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
this triggers the linter, breaks the build, to fix please add `/* eslint-disable no-underscore-dangle */` on the line right above